src/AsyncTCPVersion.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright 2016-2025 Hristo Gochkov, Mathieu Carbou, Emil Muratov
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Major version number (X.x.x) */
+#define ASYNCTCP_VERSION_MAJOR 3
+/** Minor version number (x.X.x) */
+#define ASYNCTCP_VERSION_MINOR 3
+/** Patch version number (x.x.X) */
+#define ASYNCTCP_VERSION_PATCH 2
+
+/**
+ * Macro to convert version number into an integer
+ *
+ * To be used in comparisons, such as ASYNCTCP_VERSION >= ASYNCTCP_VERSION_VAL(2, 0, 0)
+ */
+#define ASYNCTCP_VERSION_VAL(major, minor, patch) ((major << 16) | (minor << 8) | (patch))
```

Comment:
Same comments as in the WebServer version file

Reply:
same answer - will break backward compatibility.

Reply:
we settled on keeping things like this for v3 to keep backward compatibility and move to the correct naming for v4